src/unix/futex.c

```diff
@@ -57,6 +52,18 @@ static thread futex_wake_one(struct futex * f)
     return w;
 }
 
+boolean futex_wake(process p, int *uaddr)
```

Comment:
I know we already have a function called futex_wake_one(), but it might be nice to have the name of this function express the fact that at most one waiter is being woken here

Reply:
Agreed - changed to more descriptive names